seers/plugins/people_count.py

```diff
@@ -0,0 +1,125 @@
+import seer_plugin
+import seer_config
+import cv2
+import imutils
+import imutils.video
+import numpy
+import os.path
+
+class PeopleCount(seer_plugin.DataCollectorPlugin):
+	"""
+	Data collector plugin that gathers video feed
+	and uses models to determine the count of people
+	in the video stream.
+	"""
+
+	BACKGROUND_CLASS 	= "background"
+	AEROPLANE_CLASS		= "aeroplane"
+	BICYCLE_CLASS		= "bicycle"
+	BIRD_CLASS 			= "bird"
+	BOAT_CLASS 			= "boat"
+	BOTTLE_CLASS 		= "bottle"
+	BUS_CLASS 			= "bus"
+	CAR_CLASS 			= "car"
+	CAT_CLASS 			= "cat"
+	CHAIR_CLASS 		= "chair"
+	COW_CLASS 			= "cow"
+	DININGTABLE_CLASS 	= "diningtable"
+	DOG_CLASS 			= "dog"
+	HORSE_CLASS 		= "horse"
+	MOTORBIKE_CLASS 	= "motorbike"
+	PERSON_CLASS 		= "person"
+	POTTEDPLANT_CLASS 	= "pottedplant"
+	SHEEP_CLASS 		= "sheep"
+	SOFA_CLASS 			= "sofa"
+	TRAIN_CLASS 		= "train"
+	TVMONITOR_CLASS 	= "tvmonitor"
+
+	CLASSES = \
+	[
+		BACKGROUND_CLASS 	,
+		AEROPLANE_CLASS		,
+		BICYCLE_CLASS		,
+		BIRD_CLASS 			,
```

Comment:
wierd spacing?

Reply:
Nice looking spacing. Also Github likes to mess with formatting